e2e-tests/tbHttpSpec.js

```diff
@@ -48,8 +48,10 @@ describe('tbHttp', function () {
         element(by.id('submitBtn')).click();
         element(by.id('btnExp')).click();
         element(by.id('btnGet')).click().then(function(){
-            expect($('#lbGet').getText()).toBe('cancel');
-        });
+            if($('#lbGet').getText()){
```

Comment:
Ok

Get Outlook for iOS<https://aka.ms/o0ukef>




On Sun, Aug 7, 2016 at 7:06 PM -0500, "Geovanni Perez" <notifications@github.com<mailto:notifications@github.com>> wrote:


In e2e-tests/tbHttpSpec.js<https://github.com/unosquare/tubular/pull/71#discussion_r73811978>:

> @@ -48,8 +48,10 @@ describe('tbHttp', function () {
>          element(by.id('submitBtn')).click();
>          element(by.id('btnExp')).click();
>          element(by.id('btnGet')).click().then(function(){
> -            expect($('#lbGet').getText()).toBe('cancel');
> -        });
> +            if($('#lbGet').getText()){


I'm not really sure about this. Let's discuss it.

-
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/unosquare/tubular/pull/71/files/cebe1b6b6037a8648afb00b492a5146d00ee4933#r73811978>, or mute the thread<https://github.com/notifications/unsubscribe-auth/AQOKtiZKdnX198DNiPgEaZZ5RVpiamTYks5qdnMLgaJpZM4JeJIA>.